src/components/themed/ShareButtons.tsx

```diff
@@ -34,7 +34,7 @@ function ShareButton(props: { text: string; onPress: () => Promise<void>; icon:
   const [pending, handlePress] = usePendingPress(onPress)
 
   return (
-    <TouchableOpacity style={styles.button} onPress={handlePress} disabled={pending}>
+    <TouchableOpacity accessible={false} style={styles.button} onPress={handlePress} disabled={pending}>
```

Comment:
What children were made accessible here?

Reply:
The EdgeText is made accessible